models/evc.py

```diff
@@ -1125,17 +1126,27 @@ def _prepare_nni_flows(self, path=None):
             nni_flows[in_endpoint.switch.id] = flows
         return nni_flows
 
-    def _install_nni_flows(
-        self, path=None
+    def _install_unni_flows(
```

Comment:
Nice, can this be renamed to just `_install_flows`? Since now it does both uni flows and nni flows? When I first read I wasn't sure if `unni` was a typo or something else different since that's not a mef related terminology.

Reply:
Changed the name to `_install_flows`.